src/System Application/App/VS Code Integration/src/VsCodeIntegration.Codeunit.al

```diff
@@ -0,0 +1,52 @@
+namespace System.Integration;
```

Comment:
Add copyright at the top. Same for the other files.
You can find it in the other files like `UserSettings.Codeunit.al`